qutip/tests/core/test_metrics.py

```diff
@@ -396,6 +397,26 @@ def rand_super():
     ])
 
 
+@pytest.mark.parametrize('superrep_conversion',
+                         [lambda x: x, to_super, to_choi, to_kraus])
+def test__hilbert_space_dims(superrep_conversion):
```

Comment:
```suggestion
def test_hilbert_space_dims(superrep_conversion):
```
The double underscore is perhaps a bit easier to read, but we don't generally use it in QuTiP tests.